src/c++/perf_analyzer/infer_context.h

```diff
@@ -185,9 +190,20 @@ class InferContext {
   std::unique_ptr<cb::ClientBackend> infer_backend_;
   InferData infer_data_;
 
-  const bool& execute_{false};
+  const bool execute_placeholder_{
+      false};  // FIXME: update build to use C++17 instead of C++14. This is a
```

Comment:
nit: can you move the comment above line 193?

Reply:
Moved 👍